misc/ballerina-maven-plugins/docerina-maven-plugin/src/main/java/org/ballerinalang/plugin/maven/doc/DocerinaMojo.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017-2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We don't have to change this I believe. The year should stay as the year the file was added.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/9199/commits/da18aa7903275e6721a38078436f8455e87e944f